compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -606,6 +606,7 @@ class ValuePropagation : public TR::Optimization
    void transformUnknownTypeArrayCopy(TR_TreeTopWrtBarFlag *);
    void transformReferenceArrayCopy(TR_TreeTopWrtBarFlag *);
    void transformReferenceArrayCopyWithoutCreatingStoreTrees(TR_TreeTopWrtBarFlag *arrayTree, TR::SymbolReference *srcObjRef, TR::SymbolReference *dstObjRef, TR::SymbolReference *srcRef, TR::SymbolReference *dstRef, TR::SymbolReference *lenRef);
+   virtual void transformRecognizedMethod(TR::Node *node) { }
```

Comment:
This deserves some kind of doxygen comment about what it is, what it is used for, where it is called from when etc. It should probably be implemented in a cpp file rather than the header.